pywr/recorders/recorders.py

```diff
@@ -110,7 +110,29 @@ def setup(self):
         if self.nodes is None:
             self.node_names = sorted(self.model.nodes.keys())
         else:
-            self.node_names = sorted(n.name for n in self.nodes)
+            # the next line is commented because when the nodes names are provided (e.g. loaded from the json)
```

Comment:
Remove these comments and the commented code if it's not needed.

Reply:
I kept them to help checking the code modifications but I will remove them